.travis.yml

```diff
@@ -63,9 +63,9 @@ jobs:
           else
             docker run --rm -it \
             -e MAXCOMPUTE_PROJECT=$MAXCOMPUTE_PROJECT \
-            -e MAXCOMPUTE_AK=$ODPS_ACCESS_ID
-            -e MAXCOMPUTE_SK=$ODPS_ACCESS_KEY
-            -v $PWD:/work -w /work elasticdl:dev bash
```

Comment:
We didn't have this issue before?

Reply:
@workingloong I don't think we need `\` in `.travis.yml`. Please refer to SQLFlow's .travis.yml as an example.

Reply:
We skip the issue before I merge it from a branch of a forked repo.